cmd/fetch_repo/go_mod_download.go

```diff
@@ -44,34 +44,28 @@ func runGoModDownload(dl *GoModDownloadResult, dest string, importpath string, v
 
 	cmd.Stdout = buf
 	cmd.Stderr = bufErr
-	fmt.Printf("Running: %s %s\n", cmd.Path, strings.Join(cmd.Args, " "))
 	dlErr := cmd.Run()
 	if dlErr != nil {
 		if _, ok := dlErr.(*exec.ExitError); !ok {
 			if bufErr.Len() > 0 {
-				return fmt.Errorf("!%s %s: %s", cmd.Path, strings.Join(cmd.Args, " "), bufErr.Bytes())
-			} else {
-				return fmt.Errorf("!!%s %s: %v", cmd.Path, strings.Join(cmd.Args, " "), dlErr)
+				return fmt.Errorf("go mod download exec error: %s %q: %s", cmd.Path, strings.Join(cmd.Args, " "), bufErr.String())
 			}
+			return fmt.Errorf("go mod download exec error: %s %s: %v", cmd.Path, strings.Join(cmd.Args, " "), dlErr)
 		}
 	}
 
 	// Parse the JSON output.
 	if err := json.Unmarshal(buf.Bytes(), &dl); err != nil {
 		if bufErr.Len() > 0 {
-			return fmt.Errorf("3%s %s: %s", cmd.Path, strings.Join(cmd.Args, " "), bufErr.Bytes())
```

Comment:
@fmeum Do you know what this `3` and `4` (And above `!` and `!!` are coming from?

Reply:
I think that this was just an oversight, thanks for changing the messages.